### PR TITLE
[WIP] New layout using Flexbox

### PR DESCRIPTION
**Pull request by briannelson**
For #179

### CODE REVIEW

**Comment by briannelson** [comment]
@andrew @tarebyte can you (or anyone else) pull this down and give it a shot on your browser and/or ios sim? I've tested it out a bit but would love a few more 👀

**Comment by WillAbides** [comment]
This PR looked good as of this morning on chrome 55.  I didn't speak up because I didn't poke around much.

**Comment by briannelson** [comment]
I'm still figuring out a few visual bugs:

- [ ] notification subject overlaps the date on smaller screens
- [ ] notification subject overlaps entire layout on smaller screens (probably the same issue)

**Comment by tarebyte** [comment]
screen shot 2016-12-30 at 18 04 16

@briannelson looks like there is some extra padding here.

**Comment by briannelson** [comment]
@tarebyte you'll see that's the case on production as well, but if you run it through simulator, the checkboxes are lined up properly. I'll probably try to tweak the CSS a bit so that it vertically centers without having to resort to adjusting the positioning like we're doing now.

**Comment by andrew** [comment]
Just checked it out, is looking sweet! Any more to do before we can ship it?

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -30,15 +24,13 @@
   <% end %>
 
   <hr>
-
   <div class="panel panel-danger">
     <div class="panel-heading">
-      <h3 class="panel-title">Dangerzone</h3>
+      <h3 class="panel-title">Danger Zone</h3>
```
Comment (tarebyte) [inline app/views/users/edit.html.erb]: Would you mind changing this to `Danger zone` 😄

Comment (andrew) [inline app/views/users/edit.html.erb]: 🚨

Comment (briannelson) [inline app/views/users/edit.html.erb]: Oh I'm taking care of a few of those things on this branch :)

**Comment by briannelson** [comment]
I think this is mergeable.

**Comment by tarebyte** [comment]
@briannelson played with it.



:shipit:

**Pull request merged**